db/fixtures/miq_searches.yml

```diff
@@ -1255,22 +1255,22 @@
     db: Container
 - attributes:
     name: default_Project / Pod
-    description: Project / Pod
+    description: default Project / Pods
```

Comment:
Shouldn't be `Default` (capital `D` for consistency, UI is fine but I think it should be consistent here as well)